app/helpers/submission_helper.rb

```diff
@@ -17,4 +17,15 @@ def hmdmc_info
     "institution.</p>".html_safe
   end
 
+  def step_classes(step)
+    steps = { "labware" => 1, "provenance" => 2, "ethics" => 3, "dispatch" => 4 }
```

Comment:
Ideally, we wouldn't want to hardcode the steps and what order they're in here. As they're defined in `SubmissionsController` you could use [some of the methods provided by Wicked ](https://github.com/schneems/wicked#quick-reference) to get a list of wizard steps (and probably the current step)